dvc/stage/cache.py

```diff
@@ -98,6 +98,7 @@ def _create_stage(self, cache):
             repo=self.repo,
             path="dvc.yaml",
             cmd=cache["cmd"],
+            wdir=wdir,
```

Comment:
We don't need to save it into the cache itself, because stages are portable. E.g. if you run `cp foo bar` in a different wdir, it doesn't really matter as long as all paths (def_path's) make sense and deps exist.